requirements.txt

```diff
@@ -15,4 +15,4 @@ requests
 slack_bolt
 pybullet>=3.2.0
 sklearn
-pyqt5
+#pyqt5
```

Comment:
now that you've got everything installed, you can undo the change to this file (since we don't want to merge it)